### ISSUE TITLE
Calculation Matrices Get Dependencies

### ISSUE THREAD

**Issue by swongcopado** [swongcopado/dxcau2vlocity#20]
Hi Michael,
Let's take the Calculation Matrices discussion out from the other thread as I was looking at the CM that is in the trial org.
Noticed that the CM is not enabled but not sure if this is the one that you've been testing with?

![image](https://user-images.githubusercontent.com/107082678/183566624-aed1a947-c9ac-4337-92f2-e4b2d48da93f.png)

**Comment by microweii** [comment]
Ah sorry, I was working on this in Test as preparation for back-promote. And as you'd know, I can't easily back-promote this into Dev1 without making a new user story.
Can you check this in either Test or Dev2?

**Comment by swongcopado** [comment]
Just looking at this Calculation Procedure in Dev2 and just confirming if there is a relationship between the Calculation Procedure and the Calculation Matrix?  The calculation config seem to show the formula only referencing a Testvar1 that was supplied in the Calculation Procedure itself? 

![image](https://user-images.githubusercontent.com/107082678/183844133-24c2c43e-9fb8-48f9-8be0-7ad7ef8bbfe9.png)

**Comment by microweii** [comment]
Yeah that's right, this specific calculation procedure is setup standalone. Calculation procedures can be setup to make reference to a calculation matrix (which is just a lookup table), though I haven't tried deploying those two together.

**Comment by swongcopado** [comment]
OK. Yeah so can I confirm that this is no longer an issue? I believe if a Calculation Matrix is referenced in the Calculation Procedure, Get Vlocity Dependencies should pick it up. 

Let me know if there's something that I missed here.

**Comment by microweii** [comment]
> OK. Yeah so can I confirm that this is no longer an issue? I believe if a Calculation Matrix is referenced in the Calculation Procedure, Get Vlocity Dependencies should pick it up.
> 
> Let me know if there's something that I missed here.

Was wondering if you could confirm that Get Vlocity Dependencies actually does retrieve the corresponding Calculation Version and Dimension records for each as well?

I just tried in Dev2 making a CP that does a CM look-up. Then I select the CP (but not the corresponding version):
![image](https://user-images.githubusercontent.com/71686297/184268505-edbede29-bb2a-4752-9beb-3754ad5f2eee.png)

When I run the Get Vlocity Dependencies job, nothing new is selected, not even the corresponding version item seen in previous screenshot
![image](https://user-images.githubusercontent.com/71686297/184268540-74d0b890-efae-4f5b-9f47-8ddddb20a4d5.png)

This was attempted in US-0000096; CP of interest (and dependents) is CopadoTestCP_MatrixRef in Dev2.
